models/UserCartHeader.js

```diff
@@ -0,0 +1,21 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('UserCartHeader', {
+		cart_owner_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			references: {
+				model: 'User',
+				key: 'id'
+			},
+			unique: true
+		},
+		cart_status: {
```

Comment:
Again what are the statuses and how do we keep track of that so that everybody is on the same page?

Reply:
Same as Order Status, lets talk on Saturday